build.sbt

```diff
@@ -79,12 +79,14 @@ scriptedDependencies := {
 }
 
 // publish settings
-publishMavenStyle := false
-bintrayOrganization := Some("sbt")
-bintrayRepository := "sbt-plugin-releases"
-bintrayPackage := "sbt-javaagent"
-bintrayReleaseOnPublish := false
+publishMavenStyle := true
 licenses += "Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0.html")
+scmInfo := Some(ScmInfo(url("https://github.com/sbt/sbt-javaagent"), "scm:git:git@github.com:sbt/sbt-javaagent.git"))
+homepage := scmInfo.value.map(_.browseUrl)
+developers := List(
+  Developer("pvlugter", "Peter Vlugter", "peter.vlugter@lightbend.com", url("https://github.com/pvlugter"))
```

Comment:
Don't know if we have a general Lightbend team that we can use here? @ennru?

Reply:
```suggestion
  Developer("contributors", "Contributors", "https://github.com/sbt/sbt-javaagent/discussions", url("https://github.com/sbt/sbt-javaagent/graphs/contributors"))
```